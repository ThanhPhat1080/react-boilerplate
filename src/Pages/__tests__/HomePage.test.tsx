// Libraries
import * as renderer from 'react-test-renderer';

// Components
import { HomePage } from '@Pages';

describe('Testing HomePage', () => {
  test('It should render correctly', () => {
    const component = renderer.create(<HomePage />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
