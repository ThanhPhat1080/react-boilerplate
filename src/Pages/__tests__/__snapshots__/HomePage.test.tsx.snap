// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HomePage It should render correctly 1`] = `
<div>
  <h1>
    You are at
     
    <span
      aria-label="home"
      role="img"
    >
      🏠
    </span>
  </h1>
  <nav>
    <a
      href="/zoo"
      onClick={[Function]}
    >
      Go to the Zoo
       
      <span
        aria-label="tend"
        role="img"
      >
        🎪
      </span>
    </a>
  </nav>
</div>
`;
